{{ cookiecutter.package_name }}/{{ cookiecutter.module_name }}/plugin.py

```diff
@@ -7,31 +7,43 @@
 from .__about__ import __version__
 
 
-################# Configuration
-config = {
-    # Add here your new settings
-    "defaults": {
-        "VERSION": __version__,
-    },
-    # Add here settings that don't have a reasonable default for all users. For
-    # instance: passwords, secret keys, etc.
-    "unique": {
-        # "SECRET_KEY": "{{ '{{' }} 24|random_string {{ '}}' }}",
-    },
-    # Danger zone! Add here values to override settings from Tutor core or other plugins.
-    "overrides": {
-        # "PLATFORM_NAME": "My platform",
-    },
-}
+################# CONFIGURATION
+hooks.Filters.CONFIG_DEFAULTS.add_items(
```

Comment:
Subheadings for each type of config filter may help developers keep track of the parameters.

Reply:
I'm not a fan of duplicating code as comments as I find that they fall out of date & don't provide much value to the reader, so I'm going to skip this one.

Reply:
I just realized that you may have meant:
> subheadings for the type of each config filter

which IMO is not a bad idea.

I'm still going to skip this but I'm open to adding in the types in a future PR.